internal/util/plugins/api.go

```diff
@@ -0,0 +1,57 @@
+// Copyright 2020 The Operator-SDK Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+// TODO: rewrite this when plugins phase 2 is implemented.
+package plugins
+
+import (
+	"fmt"
+	"path/filepath"
+	"strings"
+
+	"sigs.k8s.io/kubebuilder/pkg/model/config"
+
+	"github.com/operator-framework/operator-sdk/internal/scaffold/kustomize"
+)
+
+// sampleKustomizationFragment is a template for samples/kustomization.yaml.
+const sampleKustomizationFragment = `## This file is auto-generated, do not modify ##
+resources:
+`
+
+// RunCreateAPI perform the SDK plugin-specific scaffolds.
+func RunCreateAPI(cfg *config.Config) error {
```

Comment:
This isn't creating any APIs, it is adding `config/samples/kustomization.yaml`:
```suggestion
func WriteSamplesKustomization(cfg *config.Config) error {
```

Reply:
done